doc/source/whatsnew/v1.0.0.rst

```diff
@@ -94,6 +94,25 @@ of the Series or columns of a DataFrame will also have string dtype.
 We recommend explicitly using the ``string`` data type when working with strings.
 See :ref:`text.types` for more.
 
+
+.. _whatsnew_1000.enhancements.unique:
```

Comment:
Note, this was chopped off of #24108 and the section is intended to be bigger, compare [here](https://github.com/pandas-dev/pandas/pull/24108/files)